### DISCUSSION TITLE
[Feature]  Add support for mysql/mariadb

### DISCUSSION THREAD

**Discussion by CountMurphy** [immich-app/immich#3434] [Feature Request]
### The feature

Hello! Is it possible to add support for other database engines, such as mariadb/mysql? Installing on baremetal and already have a mariadb instance setup. It would be nice to make use of it instead of installing a new engine.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by CountMurphy** [comment]
Found issue #562 after I posted this. I'll close it out as immich uses postgres specific queries.

**Reply by leon-v** [reply]
I always found that leaning on the features of Postgres instead of doing it in software made for very restricted code.
It would be nice to have MySQL support.
